app/Views/ScBooking/Partial/AvailableTimes/_Trial.cshtml

```diff
@@ -65,49 +66,40 @@
                     <p class="mb-3">
                         <strong>A trial date is not being booked at this stage.</strong> You are providing your
                         availability for a trial to start on <strong>one</strong> (out of a maximum of
-                        @ScMaxTrialDateSelectionsString) of your requested dates in the upcoming release.
+                        @ScMaxTrialDateSelectionsString) of your request dates for @bookingPeriodName.
```

Comment:
I just messaged Winnie to confirm "request dates" vs. "requested dates" here - there's a discrepancy in Jira vs. the wireframes.